test/WormholeL2VoteAggregator.t.sol

```diff
@@ -163,6 +163,32 @@ contract CastVote is L2VoteAggregatorTest {
     l2VoteAggregator.castVote(1, _support);
   }
 
+  // timeToProposalEnd needs to be a uint32 because the L1Block interface has a max value of uint64
```

Comment:
Why not a uint64 then? If to prevent overflow you would do `timeToProposalEnd = bound(timeToProposalEnd, 0, type(uint64).max - block.timestamp)` (maybe I have the wrong values there but you get the idea)

Reply:
Yes, true I will change it.

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/42/commits/88ecc634d0aee2651014e3ff62260019ba8601c2